### DISCUSSION TITLE
[With Backend/schedule] 일주일 간의 목표치

### DISCUSSION THREAD

**Discussion by locodingve** [slamapp/server#8] [General]
한 주간의 백앤드 목표치를 작성하여 공유하는 게시판입니다.

**Comment by locodingve** [comment]
[2022.02.21. ~ 2022.02.25] 목표치 공유 스래드 입니다.

**Reply by locodingve** [reply]
1. notification domain 수정작업
- [ ] 비즈니스 로직 test code (유닛테스트)  [40/100]
- [ ] 통합테스트 [0/100]
- [ ] merge[0/100]

2. 서버 이동
- [x] CICD 구축 (테스트) [100/100]
- [ ] CICD 구축  (운영) [0/100]
- [ ] SSL 적용 [0/0]

**Reply by KwonYeKyeong** [reply]
1. management domain api 수정
2. s3 이미지 업로드 정리해놓기 : 서버 이동 시 s3작업을 다시해야할 것을 대비

-> 이번주에 자기소개서랑 코테 준비때문에 많은 일을 하지 못할 것 같습니다. 자꾸 일을 미뤄서 죄송합니다ㅠㅠ

**Comment by locodingve** [comment]
[2022.02.28. ~ 2022.03.4] 목표치 공유 스래드 입니다.

**Reply by locodingve** [reply]
1. notification domain 수정작업
- [x] 비즈니스 로직 test code (유닛테스트)  [10/100] -> 제가 커밋을 제대로 안해서, 추가적으로 코드를 작성해야 할 것 같습니다. (기존 레포의 local 코드 존재함)
- [ ] 통합테스트 [0/100]
- [ ] merge[0/100]

2. 서버 이동
- [x] CICD 구축 (테스트) [100/100]
- [ ] CICD 구축  (운영) [0/100]
- [ ] SSL 적용 [0/0]
- [ ] 서버 배포 (3월 5일 예정)

3. was 계정 공유하기
사용자 및 권한 범위 내용은 추후 작성하겠습니다.
